support-frontend/assets/components/subscriptionCheckouts/stripeForm/stripeForm.jsx

```diff
@@ -71,255 +72,265 @@ const CardExpiryWithError = compose(withLabel, withError)(CardExpiryElement);
 const CardCvcWithError = compose(withLabel, withError)(CardCvcElement);
 const RecaptchaWithError = compose(withLabel, withError)(Recaptcha);
 
-class StripeForm extends Component<StripeFormPropTypes, StateTypes> {
-  constructor(props) {
-    super(props);
-    this.state = {
-      cardNumber: {
-        complete: false,
-        empty: true,
-        error: '',
-        errorEmpty: 'Please enter a card number',
-        errorIncomplete: 'Please enter a valid card number',
-      },
-      cardExpiry: {
-        complete: false,
-        empty: true,
-        error: '',
-        errorEmpty: 'Please enter an expiry date',
-        errorIncomplete: 'Please enter a valid expiry date',
-      },
-      cardCvc: {
-        complete: false,
-        empty: true,
-        error: '',
-        errorEmpty: 'Please enter a CVC number',
-        errorIncomplete: 'Please enter a valid CVC number',
-      },
-      cardErrors: [],
-      setupIntentClientSecret: null,
-      paymentWaiting: false,
-      recaptchaCompleted: false,
-    };
-  }
-
-
-  componentDidMount() {
-    this.setupRecurringHandlers();
-    loadRecaptchaV2();
-  }
-
-  getAllCardErrors = () => ['cardNumber', 'cardExpiry', 'cardCvc'].reduce((cardErrors, field) => {
-    if (this.state[field].error.length > 0) {
-      cardErrors.push({ field: [field], message: this.state[field].error });
+const StripeForm = (props: StripeFormPropTypes) => {
+  /**
+   * State
+   */
+  const [cardErrors, setCardErrors] = React.useState<Array<FormError<CardFieldName>>>([]);
+  const [setupIntentClientSecret, setSetupIntentClientSecret] = React.useState<Option<string>>(null);
+  const [paymentWaiting, setPaymentWaiting] = React.useState<boolean>(false);
+  const [recaptchaCompleted, setRecaptchaCompleted] = React.useState<boolean>(false);
+  const [recaptchaError, setRecaptchaError] = React.useState<FormError<'recaptcha'> | null>(null);
```

Comment:
the recaptcha error status was previously appended to the `allErrors` prop, which may have been possible when this was a class component, but is definitely not possible now it's a function. So I've added a `FormError` state for it here